docs/development/plugins/building.md

```diff
@@ -3,57 +3,16 @@ title: Building and Shipping Plugins
 linktitle: Building & Shipping
 ---
 
-## Active development / In-tree build
-
-During development however, the workflow of 1. building the plugins outside
-of the project; 2. copying to a ".plugins" folder; 3. running Headlamp pointing
-to that folder, is not a great development user-experience.
-
-While we are planning a smarter way to develop plugins outside of the project's tree
-and having them automatically built and served to the frontend, we have to
-use the work around of developing/testing them within the project (before
-building and shipping them externally once ready).
-
-For that, you can develop your plugins inside the `frontend/src/plugin/plugins` folder. E.g. if your plugin is called `MyPlugin`, then your plugin
-structure should be as follows:
-
-  ```
-  frontend/src/plugin/plugins/MyPlugin/index.tsx
-  ```
-
-## Out-of-tree build
-
-Once your plugin is working as intended, it should be compiled our of the
-Headlamp project. The `index.tsx` (or other Typescript/Javascript extensions)
-should be in its own module like:
-
-  ```
-  MyPlugin/
-    src/
-      index.tsx
-    package.json
-  ```
-
-Feel free to use `npm create` for getting the base structure/files right.
-
-For helping with compiling the plugin code we have the
-[@kinvolk/headlamp-plugin](https://www.npmjs.com/package/@kinvolk/headlamp-plugin),
-which should be installed as a dev dependency:
+This is how to start a new plugin:
```

Comment:
You have WIP in the title, not sure if it was forgotten.

Reply:
The title was fixed.